code/drv_scpi/src/scpi_sniffer/drv_scpi_node.py

```diff
@@ -16,90 +18,123 @@
 if __name__ == "__main__":
     cycler_logger = sys_log.SysLogLoggerC()
 log = sys_log.sys_log_logger_get_module_logger(__name__)
+from system_shared_tool import SysShdNodeC, SysShdIpcChanC # pylint: disable=wrong-import-position
 
 #######################          PROJECT IMPORTS         #######################
-from system_shared_tool import SysShdChanC
 
 #######################          MODULE IMPORTS          #######################
-from .drv_scpi_iface import DrvScpiHandlerC
-from .drv_scpi_cmd import DrvScipiCmdTypeE, DrvScpiCmdDataC
+from .drv_scpi_iface import DrvScpiHandlerC # pylint: disable=wrong-import-position
+from .drv_scpi_cmd import DrvScpiCmdDataC, DrvScpiCmdTypeE # pylint: disable=wrong-import-position
 
 #######################              ENUMS               #######################
+MAX_MSG = 300
+MAX_MESSAGE_SIZE = 400
 
 #######################             CLASSES              #######################
-class DrvScpiNodeC:
-    '''Returns a removable version of the DRV command.
-    '''
-    def __init__(self, tx_buffer: SysShdChanC, working_flag: Event):
-        self.__used_dev: dict = {}
-        self.working_flag: Event = working_flag
-        self.tx_buffer: SysShdChanC = tx_buffer
+class DrvScpiNodeC(SysShdNodeC):
+    "Returns a removable version of the DRV command."
+    def __init__(self, name: str, working_flag: Event, cycle_period: int):
+        '''
+        Args:
+            - working_flag (Event): Flag to know if the SCPI is working.
+            - cycle_period (int): Period of the cycle.
+        Raises:
+            - None.
+        '''
+        self.__used_dev: Dict(str, DrvScpiHandlerC) = {}
+        self.tx_scpi: SysShdIpcChanC = SysShdIpcChanC(name = 'tx_scpi',

```

Comment:
Move name as an constant at the beginning of the file.

Reply:
Done.